oem.php

```diff
@@ -92,8 +92,8 @@
 
     <div class="third">
         <h3>Get in Touch</h3>
-        <p>Talk directly with the team by emailing us at <a href="mailto:oem@elementary.io">oem@elementary.io</a>. Whether you’re an existing partner or want to explore offering elementary OS, we look forward to chatting.</p>
-        <a class="button" href="mailto:oem@elementary.io">Send an Email</a>
+        <p>Talk directly with the team by e-mailing us at <a href="mailto:oem@elementary.io">oem@elementary.io</a>. Whether you’re an existing partner or want to explore offering elementary OS, we look forward to chatting.</p>
```

Comment:
We use "email" stylistically a lot, and to my knowledge have never used "e-mail." Alternatively,

```suggestion
        <p>Talk directly with the team by sending an email to <a href="mailto:oem@elementary.io">oem@elementary.io</a>. Whether you’re an existing partner or want to explore offering elementary OS, we look forward to chatting.</p>
```